tests/ttnn/unit_tests/test_expand.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import ttnn
+import pytest
+
+
+@pytest.mark.parametrize(
+    "input_shape",
+    [
+        [1, 32],
+        [8, 1],
+        [32, 1],
+    ],
+    ids=["1d", "2d", "l2d"],
+)
+@pytest.mark.parametrize(
+    "output_shape",
+    [
+        [-1, 32],
```

Comment:
what does -1 mean it  ttnn?

Reply:
@dmakoviichuk-tt `-1` is actually from Torch. Basically means "Keep this dimension as it is" https://pytorch.org/docs/stable/generated/torch.Tensor.expand.html